pdc/apps/contact/migrations/0006_auto_20151127_0837.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contact', '0005_auto_20151027_0725'),
+    ]
+
```

Comment:
I think it also depends ('component', '0011_auto_20151126_0602'). If this migration runs before 0011, should it possibly fail because foreign key constraint?